ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -858,6 +886,11 @@ public class BottomSheetController: UIViewController {
 
         static let maxSheetWidth: CGFloat = 610
         static let minSheetWidth: CGFloat = 300
+        static let defaultMaxSheetHeight: CGFloat = 600
+        
+        // When the difference in collapsed height and max sheet height is less than this,
+        // we go into height restricted mode which skips the collapsed state and adjusts dimming.
+        static let heightRestrictedThreshold: CGFloat = 50
```

Comment:
Where did 50 come from?

Reply:
Magic number. The threshold has to be some number and this is what I came up with from testing of what distance feels sufficiently close to where a transition doesn't really make sense anymore.